SESHealth/app/src/main/res/layout/activity_packet_info.xml

```diff
@@ -1,21 +1,268 @@
+                android:layout_below="@id/packetHeightLabelTV"
+                android:text="Height"
+                android:textSize="18sp" />
+
+            <TextView
+                android:id="@+id/packetAllergiesLabelTV"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/packetHeightInfoTV"
+                android:layout_marginTop="20dp"
+                android:text="Allergies"
+                android:textSize="25sp"
+                android:textColor="#333"/>
+
+            <TextView
+                android:id="@+id/packetAllergiesInfoTV"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/packetAllergiesLabelTV"
+                android:text="Allergies"
+                android:textSize="18sp" />
+
+            <TextView
+                android:id="@+id/packetMedicationLabelTV"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/packetAllergiesInfoTV"
+                android:layout_marginTop="20dp"
+                android:text="Medication"
+                android:textSize="25sp"
+                android:textColor="#333"/>
+
+            <TextView
+                android:id="@+id/packetMedicationInfoTV"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/packetMedicationLabelTV"
+                android:text="medication"
+                android:textSize="18sp" />
+
+            <TextView
+                android:id="@+id/packetLocationLabelTV"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/packetMedicationInfoTV"
+                android:layout_marginTop="20dp"
+                android:text="Location"
+                android:textSize="25sp"
+                android:textColor="#333"/>
+
+            <TextView
+                android:id="@+id/packetLocationInfoTV"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/packetLocationLabelTV"
+                android:textStyle="bold"
+                android:textColor="#333"
+                android:text="123 University of Technology, Sydney, NSW, 2000 (Tap to open in map)"
+                android:textSize="18sp" />
+
+            <TextView
+                android:id="@+id/packetHeartbeatLabelTV"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/packetLocationInfoTV"
+                android:layout_marginTop="20dp"
+                android:text="Heart Rate"
```

Comment:
Do you think maybe we should have 'Heart Rate (BPM)' just to make it more clear?